modules/gridmap/grid_map.cpp

```diff
@@ -572,11 +572,12 @@ bool GridMap::_octant_update(const OctantKey &p_key) {
 				NavigationServer3D::get_singleton()->region_set_map(region, get_world_3d()->get_navigation_map());
 				nm.region = region;
 
+#ifdef DEBUG_ENABLED
```

Comment:
Is this related to this PR?

Reply:
This is related a clang error. I can remove since it only shows on my os.